src/main.rs

```diff
@@ -919,28 +922,52 @@ mod tests {
         );
     }
 
+    #[test]
+    fn test_function_call() {
+        parse_str(
+            r#"
+                fn test() {}
+                test()
+            "#,
+        );
+    }
+
+    #[test]
+    fn test_function_call_with_args() {
+        parse_str(
+            r#"
+                fn test(a int, b int) {}
+                test(1, 2)
+            "#
+        );
+    }
+
     #[test]
     fn test_variable() {
         parse_str(
             r#"
-                fn test() string {
-                    let foo = "1234"
-                }
+                let foo = "1234"
             "#,
         );
     }
 
     #[test]
-    fn test_variable_2() {
+    fn test_variables() {
         parse_str(
             r#"
-                fn test() string {
-                    let foo = ""
-                    let bar = foo
-                    let baz = bar
-                    return baz
-                }
+                let foo = ""
+                let bar = foo
+                let baz = bar
             "#,
         );
     }
+
+    #[test]
+    fn test_variable_string_with_spaces() {
```

Comment:
Maybe it would be also nice to add a tests for:  
- A string with `"` by using an escape char like `\` and `\\` if we gonna use the backslash as escape char
- A string with non ASCII chars like an emoji or a chineese char

Reply:
Yes, I think that these should definitely be supported. I'll add tests for the backslashes for now, and we can add more later.